notebooks/en/enterprise_cookbook_gradio.ipynb

```diff
@@ -0,0 +1,5796 @@
+{
```

Comment:
<p>It'd be better to place the "contents of app.py" and "contents of requirements.txt" under where you initially described the two files you need to edit. For example:</p><p> </p><ul><li><code>app.py</code> - This is where the demo code lives. It should look something like this:</li></ul><p>		   <code>insert code example here</code></p><ul><li><code>requirements.txt</code> - This is where we tell our Space about the libraries it will need. It should look like this:</li></ul><p>		  <code>insert txt file content</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>